### PR TITLE
add Arize Phoenix tracer to global handlers

### PR DESCRIPTION
**Pull request by RogerHYang**
# Description

Add Arize Phoenix tracer to global handlers

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [ ] Added new notebook (that tests end-to-end)
- [x] I stared at the code and made sure it makes sense

# Suggested Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: llama_index/callbacks/global_handlers.py [hunk 1]
```diff
@@ -22,6 +22,23 @@ def create_global_handler(eval_mode: str, **eval_params: Any) -> BaseCallbackHan
         handler: BaseCallbackHandler = WandbCallbackHandler(**eval_params)
     elif eval_mode == "openinference":
         handler = OpenInferenceCallbackHandler(**eval_params)
+    elif eval_mode == "openinferencetracer":
```
Comment (axiomofjoy) [inline llama_index/callbacks/global_handlers.py:25]: Should we name this `arize_phoenix` since it runs Phoenix in the background?

Comment (mikeldking) [reply llama_index/callbacks/global_handlers.py:25]: agreed, it's really an `arize_phoenix` one-click

Comment (RogerHYang) [reply llama_index/callbacks/global_handlers.py]: changed

File: llama_index/callbacks/global_handlers.py [hunk 2]
```diff
@@ -22,6 +22,23 @@ def create_global_handler(eval_mode: str, **eval_params: Any) -> BaseCallbackHan
...
+                "Please install it with `pip install -q arize-phoenix`"
+            )
+        if "exporter" not in eval_params:
+            from phoenix.trace.exporter import HttpExporter
+
+            eval_params = {
+                "exporter": HttpExporter(),
+                **eval_params,
+            }
+        handler = OpenInferenceTraceCallbackHandler(**eval_params)
```
Comment (mikeldking) [inline llama_index/callbacks/global_handlers.py:41]: should we try to detect if the phoenix server is running and print a warning if not? People might forget.

Comment (RogerHYang) [reply llama_index/callbacks/global_handlers.py:41]: as discussed offline, we'll handle this elsewhere

**Comment by logan-markewich** [comment]
Thanks @RogerHYang ! Easy PR to review 😉

**Pull request merged**